src/components/sbb-toggle-check/sbb-toggle-check.scss

```diff
@@ -0,0 +1,94 @@
+:host {
+  // only with inline-flex the component gets the right space
+  display: inline-flex;
+
+  --toggle-check-width: #{px-to-rem-build(48)};
+  --toggle-check-height: #{px-to-rem-build(28)};
+  --toggle-check-circle-diameter: #{px-to-rem-build(24)};
+  --toggle-check-circle-offset: #{px-to-rem-build(2)};
+}
+
+.toggle-check {
+  display: inline-flex;
+  align-items: center;
+  color: var(--sbb-color-charcoal-default);
+  @include text-s--regular;
+}
+
+.toggle-check__slider--after {
+  margin-inline-end: var(--sbb-spacing-fixed-3x);
+}
+
+.toggle-check__slider--before {
+  margin-inline-start: var(--sbb-spacing-fixed-3x);
+}
+
+input[type='checkbox'] {
+  position: absolute;
+  opacity: 0;
+}
+
+.toggle-check__slider {
+  position: relative;
+  cursor: pointer;
+  display: inline-block;
+  overflow: hidden;
+  min-width: var(--toggle-check-width);
+  min-height: var(--toggle-check-height);
```

Comment:
I think min-height is not necessary. But without min-width a long label with a small screensize would dominate. this would lead to a shrinking behavior for the toggle.